packages/formstr-app/src/containers/CreateFormNew/components/FormDetails/index.tsx

```diff
@@ -64,9 +83,62 @@ export const FormDetails: React.FC<FormDetailsProps> = ({
     setSavedLocally(true);
   };
 
+  const saveToMyForms = async (
+    formAuthorPub: string,
+    formAuthorSecret: string,
+    formId: string,
+    relay: string,
+    viewKey?: string
+  ) => {
+    if (!userPub) return;
+    setSavingOnNostr(true);
+    let existingListFilter = {
+      kinds: [14083],
+      authors: [userPub],
+    };
+    let pool = new SimplePool();
+    let existingList = await pool.querySync(
+      getDefaultRelays(),
+      existingListFilter
+    );
+    let forms: Tag[] = [];
+    if (existingList[0]) {
+      let formsString = await window.nostr.nip44.decrypt(
+        userPub,
+        existingList[0].content
+      );
+      forms = JSON.parse(formsString);
```

Comment:
are we sure that parse wont fail? If there is a form string which is not json, the flow would fail completely and irreparably  for the user. Should probably encapsulate in try catch and show a warning to the user on failure

Reply:
I'll handle it, this could happen when a user uses client that didn't add the array correctly